crates/transaction-pool/src/validate/eth.rs

```diff
@@ -785,14 +785,14 @@ impl<Client> EthTransactionValidatorBuilder<Client> {
     ///  - EIP-1559
     ///  - EIP-4844
     ///  - EIP-7702
-    pub fn new(client: Client) -> Self {
+    pub fn new(client: Client, local_transactions_config: Option<LocalTransactionConfig>) -> Self {
```

Comment:
This change is not required. You can use the method `with_local_transactions_config` to set the `LocalTransactionConfig`.